src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -237,78 +237,91 @@ private void HandleChangesForStaticIndexes(DatabaseRecord record, long index)
 
         private void HandleStaticIndexChange(string name, IndexDefinition definition)
         {
-            var creationOptions = IndexCreationOptions.Create;
-            var currentIndex = GetIndex(name);
-            IndexDefinitionCompareDifferences currentDifferences = IndexDefinitionCompareDifferences.None;
+            try
```

Comment:
We have HandleAutoIndexChange method, not sure if we should do the same there?

Reply:
From what i saw there is a single point where an auto index was created so i caught it closer to the call. for static index there were multiple paths